Library/Services/Implementation/NfieldSurveysService.cs

```diff
@@ -581,4 +608,9 @@ internal class UpdateSurvey
         public string InterviewerInstruction { get; set; }
     }
 
+    internal class UpdateDialMode
```

Comment:
perhaps DialModeModel would be a better name since it's used in both set and get?

Reply:
Changed the name